src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1748,6 +1748,7 @@ def aks_create(cmd, client, resource_group_name, name, ssh_key_value,  # pylint:
                kubernetes_version='',
                node_vm_size="Standard_DS2_v2",
                node_osdisk_size=0,
+               node_osdisk_diskencryptionset_id='',
```

Comment:
@MyronFanQiu - I think this introduced a bug similar to the one being fixed [here.](https://github.com/Azure/azure-cli-extensions/pull/1192)

`>az aks create -g vsoto-aks-aad-test-rg -n vsoto-aks-aad-test --enable-aad --aad-admin-group-object-id <<group id>>
Finished service principal creation[##################################]  100.0000%Property id '' at path 'properties.diskEncryptionSetID' is invalid. Expect fully qualified resource Id that start with '/subscriptions/{subscriptionId}' or '/providers/{resourceProviderNamespace}/'.

>az --version
azure-cli                         2.11.0

core                              2.11.0
telemetry                          1.0.5

Extensions:
azure-devops                      0.11.0`

Reply:
thanks, fixed by https://github.com/Azure/azure-cli/pull/14920, PTAL.